specification/domainservices/resource-manager/Microsoft.AAD/stable/2020-01-01/domainservices.json

```diff
@@ -0,0 +1,805 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DomainServiceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "HTTP 200 (OK) if the operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/DomainService"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Domain Service": {
+            "$ref": "./examples/GetDomainService.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "DomainServices"
+        ],
+        "operationId": "DomainServices_Delete",
+        "summary": "Delete Domain Service (DELETE Resource)",
+        "description": "The Delete Domain Service operation deletes an existing Domain Service.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DomainServiceNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "HTTP 202 (Accepted) if the operation was successfully started and will complete asynchronously."
+          },
+          "204": {
+            "description": "HTTP 204 (Not Content) should be used if the resource does not exist and the request is well formed."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete Domain Service": {
+            "$ref": "./examples/DeleteDomainService.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "DomainServices"
+        ],
+        "operationId": "DomainServices_Update",
+        "summary": "Update Domain Service (PATCH Resource)",
+        "description": "The Update Domain Service operation can be used to update the existing deployment. The update call only supports the properties listed in the PATCH body.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DomainServiceNameParameter"
+          },
+          {
+            "name": "domainService",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DomainService"
```

Comment:
Are all mutable properties in this request schema supported for PATCH?

Reply:
they are supported using patch as a modify domain service